internal/provider/resources/flow.go

```diff
@@ -0,0 +1,355 @@
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error creating deployment client",
+			fmt.Sprintf("Could not create deployment client, unexpected error: %s. This is a bug in the provider, please report this to the maintainers.", err.Error()),
+		)
+	}
+
+	// A deployment can be imported + read by either ID or Handle
+	// If both are set, we prefer the ID
+	var flow *api.Flow
+	if !model.ID.IsNull() {
+		var flowID uuid.UUID
+		flowID, err = uuid.Parse(model.ID.ValueString())
+		if err != nil {
+			resp.Diagnostics.AddAttributeError(
+				path.Root("id"),
+				"Error parsing Flow ID",
+				fmt.Sprintf("Could not parse Flow ID to UUID, unexpected error: %s", err.Error()),
+			)
+
+			return
+		}
+
+		flow, err = client.Get(ctx, flowID)
+	}
+
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error refreshing flow state",
+			fmt.Sprintf("Could not read Flow, unexpected error: %s", err.Error()),
+		)
+
+		return
+	}
+
+	resp.Diagnostics.Append(copyFlowToModel(ctx, flow, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	resp.Diagnostics.Append(resp.State.Set(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+// Update updates the resource and sets the updated Terraform state on success.
+func (r *FlowResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	// Unsupported - redeploy by default
+}
+
+// Delete deletes the resource and removes the Terraform state on success.
+func (r *FlowResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	var model FlowResourceModel
+
+	resp.Diagnostics.Append(req.State.Get(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	client, err := r.client.Flows(model.AccountID.ValueUUID(), model.WorkspaceID.ValueUUID())
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error creating flows client",
+			fmt.Sprintf("Could not create flows client, unexpected error: %s. This is a bug in the provider, please report this to the maintainers.", err.Error()),
+		)
+	}
+
+	flowID, err := uuid.Parse(model.ID.ValueString())
+	if err != nil {
+		resp.Diagnostics.AddAttributeError(
+			path.Root("id"),
+			"Error parsing Flow ID",
+			fmt.Sprintf("Could not parse flow ID to UUID, unexpected error: %s", err.Error()),
+		)
+
+		return
+	}
+
+	err = client.Delete(ctx, flowID)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error deleting Flow",
+			fmt.Sprintf("Could not delete Flow, unexpected error: %s", err),
+		)
+
+		return
+	}
+
+	resp.Diagnostics.Append(resp.State.Set(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+// ImportState imports the resource into Terraform state.
+func (r *FlowResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	// we'll allow input values in the form of:
+	// - "workspace_id,name"
+	// - "name"
```

Comment:
Could you please add a new directory showing examples for importing Flows?

For example, you can reference [examples/resources/prefect_variable/](https://github.com/skyscrapr/terraform-provider-prefect/tree/resource_flow/examples/resources/prefect_variable).

Reply:
changed to support ids only.

Reply:
updated as suggested